python/lsst/ts/m2com/enum.py

```diff
@@ -19,18 +19,38 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-from enum import auto, IntEnum
+from enum import auto, Enum, IntEnum
 
-__all__ = ["MsgType", "CommandStatus", "DetailedState"]
+__all__ = [
+    "MsgType",
+    "CommandStatus",
+    "DetailedState",
+    "CommandScript",
+    "PowerType",
+    "DigitalOutput",
+    "DigitalInput",
+]
+
+
+class BitEnum(Enum):
+    """Bit enum. The order is from bit 0 to bit n."""
+
+    def _generate_next_value_(self, start, count, last_values):
+        """This is an overridden function."""
```

Comment:
I updated the doc string in the commit: [8e6d00e](https://github.com/lsst-ts/ts_m2com/pull/2/commits/8e6d00e84f176e5c1227e83f56e9a124aa1afd53). This is only intensional for the digital input and output, which are specific to the circuit design based on the comments from Harris's developers. Therefore, I have no intension to extend this class to random starting value (and this might make the code more complex if someone writes several constants to fields). In the doc string, I expressed the idea that the user should just use the `auto()` . Thanks!